### PR TITLE
Add last_day function, to get last day of month

### PR DESCRIPTION
**Pull request by sdpatil**
I took care of the review comments.

### CODE REVIEW

File: presto-docs/src/main/sphinx/functions/datetime.rst [hunk 1]
```diff
@@ -96,6 +96,14 @@ Date and Time Functions
 
     Returns ``timestamp`` as a UNIX timestamp.
 
+.. function:: last_day(date) -> date
```
Comment (electrum) [inline presto-docs/src/main/sphinx/functions/datetime.rst]: The function should only be listed once in the documentation. The parameters are names, not types, so we only list multiple versions if the argument count is different. For example, we have four versions of the `month()` function, but it's only listed once.

File: presto-docs/src/main/sphinx/functions/datetime.rst [hunk 2]
```diff
@@ -374,4 +382,4 @@ Convenience Extraction Functions
 
 .. function:: yow(x) -> bigint
 
-    This is an alias for :func:`year_of_week`.
+    This is an alias for :func:`year_of_week`.
```
Comment (electrum) [inline presto-docs/src/main/sphinx/functions/datetime.rst]: What changed here? (It looks identical on GitHub)

**Comment by sdpatil** [comment]
Created a duplicate pull request by mistake will keep submitting in the original pull request https://github.com/prestodb/presto/pull/8500

File: presto-docs/src/main/sphinx/functions/datetime.rst (hunk 1 above)
Comment (sdpatil) [inline presto-docs/src/main/sphinx/functions/datetime.rst]: Made changes to remove duplicate documentation

File: presto-docs/src/main/sphinx/functions/datetime.rst (hunk 2 above)
Comment (sdpatil) [inline presto-docs/src/main/sphinx/functions/datetime.rst]: Agreed, i thought if i introduced a blank space by mistake and tried removing it but can't find the difference.

**Comment by sdpatil** [comment]
Can you please review the changes and let me know if you see any problems

**Comment by raghavsethi** [comment]
Please squash your commits and fix all whitespace issues, and then @ mention me, I'll take a look. Also do not reference issue numbers in commit messages.

**Comment by sdpatil** [comment]
Closing this pull request, because changes are moved to [Add last_day function, to get last day of month #8625](https://github.com/prestodb/presto/pull/8625)

**Pull request closed without merging**